### PR TITLE
Feature/29 market list table

### PR DESCRIPTION
**Pull request by bglownia**

### CODE REVIEW

File: libs/react-helpers/src/lib/grid-cells/price-cell.tsx [hunk 1]
```diff
@@ -17,25 +17,24 @@ export const PriceCell = ({
...
+        <span data-testid="postfix">{' ' + postfix.trim()}</span>
       )}
     </span>
-  )
-}
+  );
+};
 
-PriceCell.displayName = 'PriceCell'
+PriceCell.displayName = 'PriceCell';
```
Comment (dexturr) [inline libs/react-helpers/src/lib/grid-cells/price-cell.tsx:19]: question (non-blocking): should we be enforcing this for all components?

File: libs/react-helpers/src/lib/grid-cells/flash-cell.tsx [hunk 2]
```diff
@@ -8,84 +8,84 @@
...
 }
 
 /**
-* Get value from previous render
-*/
+ * Get value from previous render
+ */
 function usePrevious<T>(value: T): T | undefined {
-  const ref = React.useRef<T | undefined>()
+  const ref = React.useRef<T | undefined>();
```
Comment (dexturr) [inline libs/react-helpers/src/lib/grid-cells/flash-cell.tsx]: chore (non-clocking): as a follow up I think we should enable full project linting on CI, noticing a lot of files getting in with little linting errors and not sure how

File: libs/react-helpers/src/lib/grid-cells/price-cell.tsx [hunk 3]
```diff
@@ -17,25 +17,24 @@ export const PriceCell = ({
   decimalPlaces = 0,
   value,
   className = '',
-  invalid = false
+  invalid = false,
 }: IPriceCellProps) => {
-  if (!value || isNaN(Number(value)) || invalid) return <span>-</span>
-  const formattedNumber = getNumberFormat(decimalPlaces).format(value)
+  if (!value || isNaN(Number(value)) || invalid) return <span>-</span>;
```
Comment (dexturr) [inline libs/react-helpers/src/lib/grid-cells/price-cell.tsx]: minor (non-blocking): edge case here of `value = 0` will return "-" when I think we want it to return a formatted price.

**Review by dexturr — APPROVED** [review]
Minor things, nothing blocking LGTM

**Comment by dexturr** [comment]
Diff looks funky, not sure what's happeneing here but appears to have all the deal ticket changes in it as well? Maybe just needs a rebase

File: libs/react-helpers/src/lib/grid-cells/flash-cell.tsx (hunk 2 above)
Comment (bglownia) [reply libs/react-helpers/src/lib/grid-cells/flash-cell.tsx]: Agree. There is precommit hook but it's possible that some code changes will sneak to master unnoticed

File: libs/react-helpers/src/lib/grid-cells/price-cell.tsx (hunk 3 above)
Comment (bglownia) [reply libs/react-helpers/src/lib/grid-cells/price-cell.tsx]: Good point, I will add it to post merge fix list

**Pull request merged**